src/tests/ftest/util/GeneralUtils.py

```diff
@@ -50,7 +51,46 @@ def get_file_path(bin_name, dir_path=""):
     else:
         return file_path
 
+def process_host_list(hoststr):
+    """
+    This utility function takes a slurm style host string and returns a list
+    of individual hosts.
+
+    e.g. boro-[26-27] becomes a list with entries boro-26, boro-27
+
+    This works for every thing that has come up so far but I don't know what
+    all slurmfinds acceptable so it might not parse everything possible.
+    """
+
+    # 1st split into cluster name and range of hosts
+    split_loc = hoststr.index('-')
+    cluster = hoststr[0:split_loc]
+    num_range = hoststr[split_loc+1:]
+
+    # if its just a single host then nothing to do
+    if num_range.isdigit():
+        return [hoststr]
+
+    # more than 1 host, remove the brackets
+    host_list = []
+    num_range = re.sub('\[|\]', '', num_range)
```

Comment:
(lint) '\['. String constant might be missing an r prefix.

(lint) '\]'. String constant might be missing an r prefix.